pages/content.js

```diff
@@ -20,6 +20,7 @@ export async function getServerSideProps() {
 export default function Content({ currentCollections }) {
   const currentArticle = useStore((state) => state.currentArticle);
   const [popUp, setPopUp] = useState(false);
+  const [selections, setSelections] = useState([]);
   const contentRef = useRef();
```

Comment:
I can't see how a reference to your `StyledContent` component is useful to you in this page. If you're not using it, consider removing it.

Reply:
I used this to refer to the content updated by being highlighted or by removing highlights. Could u not see the whole file content.js in this PR? I m very bad with github and suck at deciding what should belong to which PR.. 😕

Reply:
Ye, sometimes it's difficult to see the overall picture when code is collapsed. If you are indeed using `contentRef`, then it's ok :)